R/plot_shinygosling.R

```diff
@@ -34,20 +34,16 @@
 #'    am_data("hg38") |>
 #'    filter(uniprot_id == "Q1W6H9") |>
 #'    to_GPos()
-#' 
-#' 
 #'
-#' # Plot the GRanges object
-#' plot_granges(gpos, title = "My GRanges Plot", subtitle = "Custom subtitle")
+#' ## Plot the GRanges object
+#' plot_granges(gpos, mode = "bar", title = "Q1W6H9 track", subtitle 
+#' = "bar plot example")
 #' }
 #'
-#' @import shiny
-#' @import shiny.gosling
-#' @import GenomicRanges
 #'
 #' @export
 plot_granges <-
-    function(as_GRanges,
+    function(gr,
```

Comment:
Here the indentation when function arguments span more than one line is (also avoid 'cryptic' names where possible)
```r
plot_granges <-
    function(
        granges,
        title = "GRanges Plot",
        subtitle = "Stacked nucleotide example"
    )
{
    ...
```

Reply:
73f173d